nano/core_test/active_transactions.cpp

```diff
@@ -983,7 +983,7 @@ TEST (active_transactions, fork_replacement_tally)
 	ASSERT_TIMELY_EQ (5s, count_rep_votes_in_election (), 8);
 
 	auto votes2 (election->votes ());
-	ASSERT_TRUE (votes2.find (nano::dev::genesis_key.pub) != votes2.end ());
```

Comment:
Leave it this as to was, iterators and equality asserts could cause problems. Leave as it was .

Reply:
Done